src/app/attendance/actions/attendanceActions.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { describe, it, vi, expect } from "vitest";
+import * as attendanceActions from "./attendanceActions";
+
+const attendanceMock = {
```

Comment:
Can we write integration test? A test where we render the `AttendanceFormBuilder` and we render inputs and verify this function is called correctly?

Reply:
Fix